### PR TITLE
entrypoint: do not force 14002 console port

### PR DESCRIPTION
**Pull request by elek**
https://github.com/storj/storagenode-docker/issues/17

### CODE REVIEW

**Review by stefanbenten — COMMENTED** [review]
This would be a backwards incompatible change, since it defaults to localhost. an ENV var specified for the container should still have precendence over the flag afaik

**Comment by elek** [comment]
> This would be a backwards incompatible change, since it defaults to localhost. 

Good point. I added an environment variable to use `0.0.0.0:14002` as default

> an ENV var specified for the container should still have precendence over the flag afaik

Unfortunately this is not the case. Flags have higher precendence....

File: docker/entrypoint [hunk 1]
```diff
@@ -61,7 +61,8 @@ if [ "${AUTO_UPDATE:-}" != "true" ]; then
   AUTO_UPDATE="false"
 fi
 
-SNO_RUN_PARAMS="${RUN_PARAMS} --console.address=:14002"
+: ${STORJ_CONSOLE_ADDRESS:=0.0.0.0:14002}
+SNO_RUN_PARAMS="${RUN_PARAMS}"
```
Comment (mniewrzal) [inline docker/entrypoint:65]: shouldn't we have here something like `${RUN_PARAMS} --${STORJ_CONSOLE_ADDRESS}`

Comment (mniewrzal) [reply docker/entrypoint:65]: or `${RUN_PARAMS} --console.address=${STORJ_CONSOLE_ADDRESS}`

Comment (mniewrzal) [reply docker/entrypoint:65]: ah I forgot that STORJ_CONSOLE_ADDRESS will be used in runtime

**Review by mniewrzal — APPROVED** [review]

**Pull request merged**